.github/workflows/e2e.yml

```diff
@@ -237,7 +237,16 @@ jobs:
 
       - name: Patch App RN
         working-directory: test/react-native/versions/${{ matrix.rn-version }}/RnDiffApp
-        run: patch --verbose --strip=0 --force --ignore-whitespace --fuzz 4 < ../../../rn.patch
+        run: |
+          patch --verbose --strip=0 --force --ignore-whitespace --fuzz 4 < ../../../rn.patch
+          ../../../rn.patch.app.js --app .
+
+      - name: Patch Android App RN
+        if: matrix.platform == 'android'
```

Comment:
```suggestion
        ${{ if: matrix.platform == 'android' }}
```

Reply:
The if statements don't have to be wrapped.

Reply:
Yes but this file already has the `if` wrapped, so its better for readability, or remove the {} from the other one.

Reply:
There was less place with than without so I removed them.